server/commands/roleplay.py

```diff
@@ -944,3 +946,87 @@ def ooc_cmd_trigger(client, arg):
         val = args[1]
         client.area.triggers[trig] = val
         client.send_ooc(f'Changed to Call "{val}" on trigger "{trig}"')
+
+
+def ooc_cmd_format_timer(client, arg):
+    """
+    Format the timer
+    Usage: /format_timer <Timer_iD> <Format>
+    """
+    args = shlex.split(arg)
+    try:
+        args[0] = int(args[0])
+    except:
+        raise ArgumentError("Timer ID should be an integer")
+    if args[0] == 0:
+        if client.is_mod or client in client.area.area_manager.owners:
+            timer = client.area.area_manager.timer
+        else:
+            client.send_ooc("You cannot change timer 0 format if you are not GM")
+            return
+    else:
+        if (
+            client.is_mod
+            or client in client.area.area_manager.owners
+            or client in client.area.owners
+        ):
+            timer = client.area.timers[args[0] - 1]
+        else:
+            client.send_ooc("You cannot change timer format if you are at least CM")
+            return
+    timer.format = args[1]
```

Comment:
maybe args[1:] cuz if you do `/format_timer 1 Time left: hh:mm` similar to the docs the command will fail unless you add ""'s